frontend/pages/admin/manage/users/_id.vue

```diff
@@ -27,6 +27,13 @@
             label="User Group"
             :rules="[validators.required]"
           ></v-select>
+          <div class="d-flex py-2 pr-2">
+            <BaseButton type="button" :loading="generatingToken" create @click.prevent="handlePasswordReset">
+              Generate Password Reset Link
```

Comment:
Can you add this as a translatable string in the en-US.json file? Probably here

https://github.com/hay-kot/mealie/blob/refactor/global-exception-handling/frontend/lang/messages/en-US.json#L698